package.json

```diff
@@ -20,12 +20,12 @@
     "eslint:docs": "eslint --fix docs/src/documentation --report-unused-disable-directives",
     "loki:approve": "yarn components loki approve",
     "loki:test": "yarn components loki test",
-    "orbit-kiwi:build": "yarn && yarn tokens build && yarn components build && yarn themer build && yarn docs build",
+    "orbit-kiwi:build": "yarn && yarn tokens build && yarn tailwind-preset build && yarn components build && yarn themer build && yarn docs build",
     "postinstall": "lerna run postinstall --stream",
     "prepare": "husky install",
     "prettier:test": "prettier --check '**/*.md'",
     "publish-packages": "zx scripts/publish.mjs",
-    "size:build": "yarn tokens build && yarn components size:build",
+    "size:build": "yarn tokens build && yarn tailwind-preset build && yarn components size:build",
```

Comment:
I do not think we need to build a tailwind package for the size

Reply:
We do. Otherwise components can't be built